apigee_sync.go

```diff
@@ -127,17 +123,23 @@ func pollChangeAgent() error {
 			log.Errorf("Get changes request failed with status code: %d", r.StatusCode)
 			switch r.StatusCode {
 			case http.StatusUnauthorized:
-				token = ""
+				tokenManager.invalidateToken()
```

Comment:
I am prefetching before expiration, but I'm not testing them in a loop or anything like that.